cmd/gendoc/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"log"
+	"os"
+	"sort"
+	"strings"
+)
+
+var configPath = flag.String("config", "", "The path to internal/config/config.go")
```

Comment:
Why not default to `internal/config/config.go`, which is probably the most used case?